src/main/java/de/vill/model/expression/CeilAggregateFunctionExpression.java

```diff
@@ -0,0 +1,46 @@
+package de.vill.model.expression;
+
+import de.vill.model.Feature;
+import java.util.Arrays;
+import java.util.List;
+import java.util.Optional;
+import java.util.Set;
+
+public class CeilAggregateFunctionExpression extends AggregateFunctionExpression {
+    private final String featureName;
+
+    public CeilAggregateFunctionExpression(final String featureName) {
+        super(featureName, null);
+        this.featureName = featureName;
+    }
+
+    @Override
+    public String toString(final boolean withSubmodels, final String currentAlias) {
+        return super.toString(withSubmodels, "len", currentAlias);
+    }
+
+    @Override
+    public List<Expression> getExpressionSubParts() {
+        return Arrays.asList();
```

Comment:
Inconsistent to other methods which use Collections.emptyList() is this intended?